src/package.json

```diff
@@ -28,8 +29,11 @@
     "@nestjs/core": "^9.0.0",
     "@nestjs/mapped-types": "^1.2.2",
     "@nestjs/platform-express": "^9.0.0",
+		"@nestjs/schedule": "^3.0.1",
     "@sentry/node": "^7.52.1",
     "atob": "^2.1.2",
+		"aws-sdk": "^2.1454.0",
+		"aws-sdk-js-codemod": "^0.17.7",
```

Comment:
this codemod may not be needed, remove and rechk once

Reply:
I removed it.